moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -488,8 +488,8 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
     const std::string& eef = end_effector_link.empty() ? end_effector_link_ : end_effector_link;
 
     // if multiple pose targets are set, return the first one
-    std::map<std::string, std::vector<geometry_msgs::PoseStamped> >::const_iterator jt = pose_targets_.find(eef);
-    if (jt != pose_targets_.end())
+    const auto jt = pose_targets_.find(eef);
```

Comment:
when running clang-tidy across the codebase, i believe we decided (@BryceStevenWilley ) not to replace explicit iterators with ``auto``, by disabling that change. Having it explicit greatly improves readability IMHO, and wastes the original author's effort in making it easier to read. Could you revert these types of changes please?

Reply:
Seeing how it has been decided one way I will revert them back. But just for the record, I personally find it impossible to argue in favor of explicit iterators.

Reply:
I don't want to restart a whole debate on what's more readable, but IMHO this type of change _would_ improve readability. IDE's can do code completion or show type info, if needed. I guess I'm just surprised to be in the minority.